polaris-react/src/utilities/event-listener/hooks.ts

```diff
@@ -0,0 +1,21 @@
+import {useEffect, useRef} from 'react';
+
+export type ElementType = Window | Document | HTMLElement;
+
+export function useEventListener(
+  event: string,
+  handler: () => void,
```

Comment:
```suggestion
  handler: (event: Event) => void,
```

While this isn't the most informative type it at least indicates an event parameter is present.

Reply:
Nice catch. I missed this from the previous implementation 👍